### PR TITLE
Added ImGui as a submodule

### PR DESCRIPTION
**Pull request by OsiJr**
[AB#119](https://dev.azure.com/euclideon/57c3bd2a-8f94-4578-b195-2e4fa9d8f295/_workitems/edit/119)

### CODE REVIEW

File: src/vcConvert.cpp [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #include "vcConvert.h"
 
 #include "imgui.h"
+#include "imgui_internal.h"
```
Comment (samsinsane) [inline src/vcConvert.cpp]: No comment for what requires this?

File: src/vcModals.cpp [hunk 2]
```diff
@@ -16,6 +16,8 @@
 #include "udStringUtil.h"
 
 #include "imgui.h"
+#include "imgui_internal.h"
```
Comment (samsinsane) [inline src/vcModals.cpp]: No comment for what requires this?

File: src/vcConvert.cpp (hunk 1 above)
Comment (OsiJr) [reply src/vcConvert.cpp]: `SeparatorEx` in case people look at this in future

File: src/vcModals.cpp (hunk 2 above)
Comment (OsiJr) [reply src/vcModals.cpp]: Lots of stuff in this file

**Pull request merged**